weblate/accounts/models.py

```diff
@@ -496,6 +496,30 @@ class Profile(models.Model):
         default=False
     )
 
+    DASHBOARD_SUBSCRIPTIONS = "your-subscriptions"
+    DASHBOARD_LANGUAGES = "your-languages"
+    DASHBOARD_ALL = "projects"
+    DASHBOARD_COMPONENT_LIST = "list"
+    DASHBOARD_CHOICES = (
+        (DASHBOARD_SUBSCRIPTIONS, _('Your subscriptions')),
+        (DASHBOARD_LANGUAGES, _('Your languages')),
+        (DASHBOARD_ALL, _('All projects')),
+        (DASHBOARD_COMPONENT_LIST, _('Component list')),
+    )
+    dashboard_view = models.CharField(
+        max_length=100,
+        choices=DASHBOARD_CHOICES,
+        verbose_name=_('Default dashboard view'),
+        default=DASHBOARD_SUBSCRIPTIONS,
+    )
```

Comment:
We use integer before and decided switched to string when we need to get slug from user.profile. I don't want to have mapping between object/integers and slug in two different files (index.html and that where I set active_tab_slug). 
Do you have idea how to avoid two mapping and use integers in models.py?